client/src/routes/widgetModule/WidgetModule.tsx

```diff
@@ -95,6 +125,188 @@ export const WidgetModule: React.FC<Props> = observer(({ domainName }) => {
     setFormFields({ ...formFields, widgetValue: '' })
   }
 
+  const createAlias = async (alias: string, forward: string) => {
+    setProcessStatus({
+      type: ProcessStatusTypes.PROGRESS,
+      render: (
+        <BaseText>
+          {walletStore.isMetamaskAvailable
+            ? 'Waiting for a transaction to be signed'
+            : 'Sign transaction on mobile device'}
+        </BaseText>
+      ),
+    })
+
+    try {
+      const numAlias = await rootStore.easClient.getNumAlias(domainName)
+      const maxAlias = await rootStore.easClient.maxNumAlias()
+      const publicAliases = await rootStore.easClient.getPublicAliases(
+        domainName
+      )
+      console.log('### maxNum', numAlias)
+      console.log('### numAlias', numAlias)
+      console.log('### publicAliases', publicAliases)
+
+      if (numAlias >= maxAlias) {
+        setProcessStatus({
+          type: ProcessStatusTypes.PROGRESS,
+          render: (
+            <BaseText>
+              {walletStore.isMetamaskAvailable
+                ? 'Waiting for a transaction to be signed'
+                : 'Sign transaction on mobile device'}
+            </BaseText>
+          ),
+        })
+
+        const removingAlias = publicAliases[0]
+
+        const newPublicAliases = publicAliases.filter(
+          (item) => item !== removingAlias
+        )
+
+        const setResult = await rootStore.easClient.setPublicAliases({
+          domainName,
+          aliases: newPublicAliases,
+          onTransactionHash: () => {
+            setProcessStatus({
+              type: ProcessStatusTypes.PROGRESS,
+              render: <BaseText>Waiting for transaction confirmation</BaseText>,
+            })
+          },
+        })
+
+        if (setResult.error) {
+          throw setResult.error
+        }
+
+        const delResult = await rootStore.easClient.deactivate({
+          domainName,
+          alias: removingAlias,
```

Comment:
alias is bytes32, not string

Reply:
it works well, i checked [the repo](https://github.com/polymorpher/eas/blob/a226a3a3d0f8375d702eab1abfbe9e42ed86a538/client/src/Home.tsx#L216) and there is also a string

Reply:
okay, i missed the part where it converts to bytes32 in client code